.gitignore

```diff
@@ -329,3 +329,4 @@ ASALocalRun/
 # MFractors (Xamarin productivity tool) working folder 
 .mfractor/
 /Ginger/GingerCore/Plugins/Jenkins/gingerPlugin/work
```

Comment:
what is this ".getignore" file? why it is part of PR?

Reply:
Done